### ISSUE TITLE
[Others] Midterm_#Q8

### ISSUE THREAD

**Issue by Daegeo** [goldshakil/ElectricalProgrammingCourse2020#140]
To implement x^(1/y), I used  binomial series which is uploaded in Wekipedia(Professor uploaded). However, It converges for |x|<1 for any real or complex number α. so if x is bigger than 2, it can not be used. Should I use other Taylor series?. I don't know how to solve it. Thank you for reading my question.

**Comment by goldshakil** [comment]
Yes there are many other taylor series which have a better implementation. 

Please sesrch for them
